low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -169,22 +173,41 @@ def __init__(self, unit):
         self.previous_substate = "N/A"
         self.previous_pid = "N/A"
         self.nonactive_enter_timestamp = time.time()
+        self.active_enter_timestamp = time.time()
         self.nonactive_threshold = int(
             Conf.get(SSPL_CONF,
                      f"{ServiceMonitor.name().upper()}>threshold_inactive_time",
                      '60'))
+        self.active_threshold = int(Conf.get(SSPL_CONF,
+                     f"{ServiceMonitor.name().upper()}>threshold_active_time",
+                     '30'))
         self.properties_changed_signal = None
         self._service_state = ActiveState
         self._unit_state = None
 
     @property
     def is_enabled(self):
-        return 'disabled' not in self.properties_iface.Get(
-            UNIT_IFACE, 'UnitFileState')
+        path_array = self.properties_iface.Get(SERVICE_IFACE, 'ExecStart')
+        # check if service is configured on system,
+        # if yes then check if it is in enabled state.
+        try:
+            command_line_path = str(path_array[0][0])
+        except IndexError as err:
+            logger.error(
+                "Unable to find %s path due to %s" % (self.name, err))
+            command_line_path = "NA"
+        if command_line_path != "NA":
+            return 'disabled' not in self.properties_iface.Get(
+                UNIT_IFACE, 'UnitFileState')
+        else:
```

Comment:
`return False` can be outside of else as well.

Reply:
done